readme.md

```diff
@@ -556,6 +556,7 @@
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 - [Corda](https://github.com/chainstack/awesome-corda#readme) - Open source blockchain platform designed for business.
 - [Waves](https://github.com/msmolyakov/awesome-waves#readme) - Open source blockchain platform and development toolset for Web 3.0 apps and decentralized solutions.
+- [Hive](https://github.com/ledgerconnect/awesome-hive#readme) - Powerful blockchain for Web 3.0 apps.
```

Comment:
"Web 3.0" is market jargon. Make the description more useful.

Reply:
@sindresorhus ok, updated.